test/function/samples/external-alias-parent/_config.js

```diff
@@ -0,0 +1,36 @@
+var assert = require( 'assert' );
+var path = require( 'path' );
+
+module.exports = {
+	description: 'includes an external module included dynamically by an alias',
+	options: {
+		input: path.join( __dirname, 'first', 'main.js' ),
+		external: function ( id, parentId, isResolved ) {
+			if ( isResolved === false || !parentId )
+				return;
```

Comment:
Maybe `return false` would be a little clearer here?

Reply:
Would it be possible to also have a test where it is essential that the resolution is done pre-resolve? This would also be helpful as documentation for the future.

Reply:
I'm struggling to come up with a good example. I can only think of scenarios where its essential for it to be post-resolve so that the external module can be resolved, while not returning true for external before it is resolved.

Reply:
I can create another example for that, if that sounds sensible?